### PR TITLE
Fix command with --force and --drop

### PR DESCRIPTION
**Pull request by WouterJ**
#241 was not backported, while the `--force` option was backported. This resulted in errors.

Meanwhile, I also fixed the script to also work with doctrine/dbal 2.4 (in which case the `TableExistsException` did not yet exists).

### CODE REVIEW

**Comment by WouterJ** [comment]
@dbu can you please merge 1.1 into master (so we have support for dbal 2.4 in master too?)

File: src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php [hunk 1]
```diff
@@ -69,16 +71,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-                            $output->writeln($e->getMessage());
+                try {
+                    foreach ($schema->toDropSql($connection->getDatabasePlatform()) as $sql) {
+                        if (true === $input->getOption('dump-sql')) {
+                            $output->writeln($sql);
+                        } else {
+                            try {
+                                $connection->exec($sql);
+                            } catch (\Exception $e) {
+                                $output->writeln($e->getMessage());
```
Comment (dbu) [inline src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php]: ups, did we really want to merge this like this? why would we then even care about catching the DBALException below?

**Comment by dbu** [comment]
there is a merge conflict with master. i can port the changes, but guess we first need to sort out if you intended the code as it is now.

File: src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php (hunk 1 above)
Comment (WouterJ) [inline src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php]: Hmm, indeed. Seems like this PR does something wrong and should be revered. However, we should change the try..catch statement in master to be the same as this one.

**Pull request merged**